roadmap.md

```diff
@@ -23,26 +23,40 @@ implement the spec.
 1. Create and deploy a website that features a simple overview, email list and directs visitors to Github.
 1. Store all website assets in the CloudEvents repository, under the governance
 of the working group.
-
-*0.2*
-
 1. Have at least 2 implementations of the specification that can demonstrate interoperability.
 1. Include a specification for mapping the CloudEvents specification to HTTP.
 1. Include a specification for mapping the CloudEvents specification to JSON.
 1. Changes to the spec to facilitate adoption.
-1. Draft documentation and developer guide.
 1. Publicize at conferences ([CloudNativeCon Europe](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/)).
-
-*0.3*
-
 1. Interoperability demo.
     1. At least one open source implementation of sending and receiving events.
     1. Events are sent by code written by Developer1 and recieved by code written by Developer2, where Developer1 has no knowledge of Developer2.
     1. Draft trigger specification (e.g. how events + actions are associated).
-1. Draft documentation and user guides.
+
+*0.2*
+
+1. Incorporate learnings and feedback from interop demo to support wider
+   adoption.
+1. Draft documentation, developer and/or user guide.
+1. Resolve all known large design issues (excluding security issues)
+1. Resolve all known "proposed required attributes" issues
+1. Interoperability demo 2
+    1. Details to be determined
+	1. Showcase demo at conferences - perhaps KubeCon NA 2018
+1. Define the set of protocol and serialization mappings we're going to
+   produce for 0.4 milestone
+
+*0.3*
+
+1. Resolve all known "proposed optional attributes" issues
```

Comment:
I'm not convinced that this needs to be tied to a particular milestone -- from my perspective security and authorization trump this point by a long margin.

Reply:
All work will continue to be done based on when PRs are submitted. So, if PRs for later milestones are submitted prior to use entering that milestone's timeframe they'll still be looked at. As for security itself, suggest an alternative milestone to put those items but for now I assuming trying to get the core required attributes defined was the most important thing. And note that if we think we need a required security attribute then that would fall into that category.